seqr/utils/elasticsearch/es_search.py

```diff
@@ -82,6 +82,20 @@ def __init__(self, families, previous_search_results=None, inheritance_search=No
         self._no_sample_filters = False
         self._any_affected_sample_filters = False
 
+    @staticmethod
+    def _parse_xstop(result):
+        xstop = result.pop(XSTOP_FIELD, None)
+        if xstop:
+            end_chrom, end = get_chrom_pos(xstop)
+            if end_chrom != result['chrom'] or end != result['end']:
```

Comment:
checking if either is different means that it is easy to have endChrom defined for all translocations even if they are within the same chromosome. I think it is okay because for translocations we always want to show endChrom, for complex variants END2 is only defined in cases we would want to show it, and for insertions I added the special case handling below anyways